lib/ui/user_interface.rb

```diff
@@ -11,6 +11,24 @@ def welcome
     display_board(grid)
   end
 
+  def who_goes_first?
+    begin
+      puts "\nWho should go first?\nPress [1] for you.\nPress [2] for the computer.\nChoice: "
+      selection = gets.chomp
+      validate_who_goes_first(selection)
+    rescue => error
+      puts error
+      retry
+    end
+    return selection
+  end
+
+  def validate_who_goes_first(selection)
+    unless(selection == '1' || selection == '2')
```

Comment:
These are a little more hidden in the UI, but hopefully more descriptive in the initializer now.  They pull values from a hash in the script now.

[Initializer](https://github.com/saralein/tic-tac-toe_ruby/pull/7/files#diff-6bfad8daa181b5e0a43f92245aa59e75R19)

[Script](https://github.com/saralein/tic-tac-toe_ruby/pull/7/files#diff-8c6067d477dcdcf59cd7a3e06b5c7405R6)

[UI](https://github.com/saralein/tic-tac-toe_ruby/pull/7/files#diff-2bad3e77eef91be4afbbff5bceebfe6dR38)